.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Shukant Pal
+# SPDX-License-Identifier: Apache-2.0
+name: Publish to PyPI
+
+on:
+  workflow_dispatch:
```

Comment:
We only want to publish when we make a release.

When is workflow_dispatch set?

Reply:
You can manually trigger this action here: https://github.com/KhronosGroup/KTX-Software/actions